### PR TITLE
Merge upstream changes up to v2.2.1

### PR DESCRIPTION
**Pull request by kswilster**
[[NEW-6760]](https://salsify.atlassian.net/browse/NEW-6760) Pulling in upstream changes to fix [this bug](https://github.com/cibernox/ember-power-select/blob/master/CHANGELOG.md#2012)

### CODE REVIEW

**Comment by kswilster** [comment]
[this comparison](https://github.com/cibernox/ember-power-select/compare/master...salsify:merge-upstream/v2.2.1) between upstream master and this branch is probably useful. This PR is just maintaining the differences between upstream master and our fork

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salsify/ember-power-select",
-  "version": "2.0.8-beta.0-0.3.2",
+  "version": "2.2.1-0.3.2",
```
Comment (mnoble01) [inline package.json:3]: Can we drop the `-0.3.2` as [ember-power-select does](https://github.com/cibernox/ember-power-select/blob/master/package.json#L3)?

**Review by mnoble01 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (kswilster) [reply package.json:3]: this is our fork version. It indicates the upstream version as well as the nature of our changes on top of upstream. [here's an explanation](https://github.com/semver/semver/issues/17#issuecomment-49895206) of a similar versioning scheme

Comment (kswilster) [reply package.json:3]: Actually [here's the explanation](https://salsify.atlassian.net/wiki/spaces/ENG/pages/96147257/Semantic+Versioning+for+Forks) for our versioning scheme for this fork.

**Pull request merged**